### DISCUSSION TITLE
Export Data by Class Name Instead of by Individual?

### DISCUSSION THREAD

**Discussion by alexneuf** [cnr-isti-vclab/TagLab#38] [Q&A]
New TagLab user here, so forgive me if this is common knowledge.

I've imported a custom library and after tagging my corals, I would like to export data by class name rather than individual colony. So if I have 20 corals of Species X and 20 corals of Species Y, how do I export two .csv rows that "sum" the 20 colonies of each species, giving me population data on all of Species X and Y, rather than 40 .csv rows that each give me the data for just one individual colony?

Many thanks!

**Comment by maxcorsini** [comment]
Hi Alexneuf, 

The export functionalities are aware of the visibility of the classes. So, you can make visible the corals of colonies X only and export CSV data, then make visible the corals of species Y only and export them as another table. 

I hope this solve your problem.

All the best

**Comment by alexneuf** [comment]
Hi, thanks for the response.

I don’t think that will solve my problem, unfortunately. I’ve tried to turn of the visibility of genotypes Y and Z and then export data for genotype X, which is still visible. This gives me a .csv of ~50 rows (one for each colony identified). Is there a way to export a .csv with one row, that sums all the area data for those 50 colonies together into one large number?

Alex Neufeld
Photomosaics and Technology Coordinator
Phone: (786) 780-2661

Coral Restoration Foundation™
coralrestoration.org<http://www.coralrestoration.org/>

Headquarters
89111 Overseas Hwy, Tavernier, Florida 33070

Exploration Center
5 Seagate Blvd, Key Largo, Florida 33037

Phone
(305) 453-7030

INSTAGRAM @coralrestorationfoundation<https://www.instagram.com/coralrestorationfoundation/>
FACEBOOK CoralRestorationFoundation<http://facebook.com/CoralRestorationFoundation>
TWITTER @coralcrf<http://twitter.com/coralcrf>

From: Massimiliano Corsini ***@***.***>
Date: Friday, April 22, 2022 at 9:05 AM
To: cnr-isti-vclab/TagLab ***@***.***>
Cc: Alex Neufeld ***@***.***>, Author ***@***.***>
Subject: Re: [cnr-isti-vclab/TagLab] Export Data by Class Name Instead of by Individual? (Discussion #38)

Hi Alexneuf,

The export functionalities are aware of the visibility of the classes. So, you can make visible the corals of colonies X and export CSV data, then make visible the corals of species Y and export them as another table.

I hope this solve your problem.

All the best
